iotedgedev/modules.py

```diff
@@ -155,7 +155,7 @@ def build_push(self, no_build=False, no_push=False):
             key = (module_name, platform)
             if key in image_tag_map:
                 tag = image_tag_map.get(key)
-                replacements["${{MODULES.{0}.{1}}}".format(module_name, platform)] = tag
+                replacements[module_name] = tag
```

Comment:
one question here. If the module didn't use this schema, will it also be replaced?

Reply:
No, see https://github.com/Azure/iotedgedev/blob/master/iotedgedev/deploymentmanifest.py#L84-L100

Reply:
Can we use regex to get the to be process modules?